int16 d1pwm1
int16 d1pwm2
int16 d2pwm1
int16 d2pwm2
bool got_ball
